src/js/components/login.js

```diff
@@ -6,10 +6,13 @@
 */
 import classnames from 'classnames';
 import FluxComponent from 'flummox/component';
+import connectToStores from 'flummox/connect';
 import fluxMixin from 'flummox/mixin';
 import React from 'react';
 import Url from 'urlgray';
 
+import LoginForm from './handlers/loginform';
```

Comment:
Yeah, I changed my tune halfway through, got thrown off by you naming `/fxa-authorize` as `login`.